src/OrchardCore.Modules/OrchardCore.Security/SecurityHeaderDefaults.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using OrchardCore.Security.Options;
+
+namespace OrchardCore.Security
+{
+    public static class SecurityHeaderDefaults
+    {
+        internal static readonly string PoliciesSeparater = ", ";
```

Comment:
```suggestion
        internal static readonly char[] PoliciesSeparater = new [] { ',', ' ' };
```

Reply:
Then rename to `PoliciesSeparators` (added the o and the s)

Reply:
Oops, typo ;)

Reply:
Permissions policies separators is not similar to CSP

Reply:
Seems `;` can be used for both